roles/vpn/templates/etc_openvpn_server.conf.j2

```diff
@@ -298,9 +299,18 @@ status openvpn-status.log
 # 4 is reasonable for general usage
 # 5 and 6 can help to debug connection problems
 # 9 is extremely verbose
-verb 3
+verb {{ openvpn_verb }}
 
 # Silence repeating messages.  At most 20
 # sequential messages of the same message
 # category will be output to the log.
 ;mute 20
+
+# Mask your settings with mtu
```

Comment:
Would you mind adding a bit more explanation, like "Your mtu can be used by others to detect VPN use", "Mask your settings by using a lower mtu".

Thanks again for this patch.